gaurun/global.go

```diff
@@ -15,10 +15,12 @@ var (
 	QueueNotification chan RequestGaurunNotification
 	// TLS certificate and key for APNs
 	CertificatePemIos CertificatePem
+	// AuthKey is the result of loading the .p8 certificate
+	AuthKey *ecdsa.PrivateKey
```

Comment:
This global variable name does not have enough information volume instead of importance. Append something prefix to specify role.